tests/vcr_python_wrapper.py

```diff
@@ -159,7 +159,7 @@ def filter_request_uri(request):
     if test_params['test_name'] == 'host':
         fam_vcr.register_matcher('host_body', host_body_matcher)
         body_matcher = 'host_body'
-    elif test_params['test_name'] in ['subscription_manifest', 'manifest_role']:
+    elif test_params['test_name'] in ['subscription_manifest', 'manifest_role', 'basic_setup_rhel7_role']:
```

Comment:
:+1: By reading your comment, i expected a much more involved change here. LOL

Reply:
But if i read it correctly, the manifest will still be recorded. It's just not considered, when matching the requests again.
So be sure not to use a valuable manifest.

Reply:
Damn, you're right. I somehow thought it's not recorded then, but no.

We'd need to adjust https://github.com/theforeman/foreman-ansible-modules/blob/a071f171f6aa178545ec884657bc1e0511005505/tests/vcr_python_wrapper.py#L113-L115 to make it really unrecorded.

@wbclark I can look at this later this week in more detail if you want to?

Reply:
And twitter never fails to amaze me with timing, so: https://twitter.com/IanColdwater/status/1358805476616544258